addons/storyshots/src/test-body-image-snapshot.js

```diff
@@ -45,14 +55,13 @@ export const imageSnapshot = ({
         throw e;
       })
       .then(() => beforeScreenshot(page, { context, url }))
-      .then(() =>
-        page.screenshot().then(image => {
-          expect(image).toMatchImageSnapshot(getMatchOptions({ context, url }));
-        })
-      );
+      .then(() => page.screenshot(getScreenshotOptions({ context, url })))
+      .then(image => {
+        expect(image).toMatchImageSnapshot(getMatchOptions({ context, url }));
+      });
   };
 
-  testFn.beforeEach = () =>
+  testFn.beforeAll = () =>
```

Comment:
Moving this to `beforeAll` is a perf booster. No need to re-run Chrome for each tests.